configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
one last thing, @pwolfram: if we do not want to use maxChunkSize by default, shouldn't we comment out maxSizeChunk in the MOC section in the various config files?
sorry if I keep going on this: it is clear that I don't have the full picture clear..